<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1067</width>
    <height>811</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>100000</width>
    <height>100000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>141</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>串口设置</string>
       </property>
       <widget class="QComboBox" name="portBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="connectButton">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>60</y>
          <width>71</width>
          <height>25</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1000</width>
          <height>1000</height>
         </size>
        </property>
        <property name="text">
         <string>连接</string>
        </property>
       </widget>
       <widget class="QLabel" name="showConnectLabel">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>38</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="res.qrc">:/icon/connect_fail</pixmap>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>181</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="loopGroupBox">
       <property name="title">
        <string>闭环控制</string>
       </property>
       <widget class="QRadioButton" name="positionLoopRadioButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>115</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>位置环</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="speedLoopRadioButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>115</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>速度环</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>0</y>
      <width>311</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QGroupBox" name="motorGroupBox">
       <property name="title">
        <string>电机</string>
       </property>
       <widget class="QRadioButton" name="yawRadioButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>115</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>云台俯仰</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="chassis1RadioButton">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>30</y>
          <width>115</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>底盘电机1</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="pitchRadioButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>115</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>云台偏航</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="chassis2RadioButton">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>30</y>
          <width>115</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>底盘电机2</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="chassis3RadioButton">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>70</y>
          <width>115</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>底盘电机3</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="chassis4RadioButton">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>70</y>
          <width>115</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>底盘电机4</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>0</y>
      <width>321</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QGroupBox" name="paramGroupBox">
       <property name="title">
        <string>PID调参</string>
       </property>
       <widget class="QLabel" name="pLabel">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>30</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>P</string>
        </property>
       </widget>
       <widget class="QLabel" name="iLabel">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>60</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>I</string>
        </property>
       </widget>
       <widget class="QLabel" name="dLabel">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>90</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>D</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="pSpinBox">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>30</y>
          <width>71</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="iSpinBox">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>60</y>
          <width>71</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="dSpinBox">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>90</y>
          <width>71</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="writePushButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>41</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>写入</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>10</y>
          <width>31</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>提示</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="hintText">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>30</y>
          <width>151</width>
          <height>81</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>1041</width>
      <height>561</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="4,4">
     <item>
      <widget class="QWidget" name="positionWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="speedWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="controlPlotButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>暂停</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>gridLayoutWidget</zorder>
   <zorder>gridLayoutWidget_2</zorder>
   <zorder>controlPlotButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1067</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
